### PR TITLE
[autoscaler] [tune] More doc fixes

### PR DESCRIPTION
**Pull request by ericl**

### CODE REVIEW

**Comment by richardliaw** [comment]
@vlad17 didn't you also need `dpkg configure -a` after `sudo pkill -9`?

**Comment by vlad17** [comment]
@richardliaw yeah

File: python/ray/autoscaler/aws/development-example.yaml [hunk 1]
```diff
@@ -80,6 +81,9 @@ file_mounts: {
 
 # List of shell commands to run to set up nodes.
 setup_commands:
+    # Consider uncommenting these if you run into dpkg locking issues
+    # - sudo pkill -9 apt-get || true
+    # - sudo pkill -9 dpkg || true
```
Comment (richardliaw) [inline python/ray/autoscaler/aws/development-example.yaml]: why not also 
`- sudo dpkg --configure -a` here?

Comment (ericl) [reply python/ray/autoscaler/aws/development-example.yaml]: Done

File: doc/source/autoscaling.rst [hunk 2]
```diff
@@ -31,7 +31,7 @@ To run connect to applications running on the cluster (e.g. Jupyter notebook) us
 
 .. code-block:: bash
 
-    $ ssh -L 8899:localhost:8899 -i <key> <user>@<addr> jupyter notebook --port=8899
+    $ ssh -L 8899:localhost:8899 -i <key> <user>@<addr> 'source ~/anaconda3/bin/activate tensorflow_p36 && jupyter notebook --port=8899`
```
Comment (richardliaw) [inline doc/source/autoscaling.rst]: does this work? the `example.yaml` doesn't install in this env right?

**Comment by richardliaw** [comment]
@ericl sphinx lint failed

**Comment by ericl** [comment]
Also added a full mnist example for hyperband (requested by user).

**Review by richardliaw — APPROVED** [review]
Looks good; would probably be nice to put a smoke test in for the example

**Comment by ericl** [comment]
Good though, added smoke test

**Comment by ericl** [comment]
Test hang looks unrelated, merging

**Pull request merged**